e2e/src/tests/fileBrowserPage.test.js

```diff
@@ -0,0 +1,19 @@
+import { vBtn, vListItemSubtitle, vList } from 'jest-puppeteer-vuetify';
+import {
+  waitForRequestsToFinish,
+} from '../util';
+
+describe('file browser page', () => {
+  it('view file browser', async () => {
+    await expect(page).toClickXPath(vBtn('Public Dandisets'));
+    await waitForRequestsToFinish();
+
+    // Assert that the page has loaded with a list of dandisets
+    await expect(page).toClickXPath(vListItemSubtitle('DANDI:000001'));
+    await expect(page).toClickXPath(vBtn('Files'));
+    await waitForRequestsToFinish();
+
+    // Ensure that the page has loaded with a folder
+    await expect(page).toContainXPath(vList());
```

Comment:
I _think_ something like this should work, although I haven't tested it:
```suggestion
    // Ensure that the page has loaded with a folder
    await expect(page).toContainXPath(vListItem('foo'));
```

That way there's a stricter test for the presence of the folder, rather than assuming that the presence of a `v-list` element means the folder component is there.

Reply:
changed in 21e7b0f81fed2209d602daa987c6657bce2d1bf0